### DISCUSSION TITLE
Add function to get branch name

### DISCUSSION THREAD

**Discussion by JaroslawSlabik** [airblade/vim-gitgutter#800] [Ideas]
Hello,
Would it be a good idea to add a function that returns the name of the branch? 
This function would be called `GitGutterGetBranchName()`.

What do you think about this idea?

**Comment by airblade** [comment]
The aim of the plugin is to ease working with diffs and hunks.  Showing the branch name is out of scope.

I assume this is for your status line.  If you use a status line plugin, it probably already supports this; if you construct your own status line, adding the branch name would be straightforward.
